src/mocks/mockRegisterResponse.ts

```diff
@@ -0,0 +1,4 @@
+export const mockRegisterResponse = {
```

Comment:
Is this used anywhere?

Reply:
Yes in my test file that mocks the ApiClient